mslice/cli/_mslice_commmands.py

```diff
@@ -125,42 +136,53 @@ def Slice(InputWorkspace, Axis1=None, Axis2=None, NormToOne=False):
     x_axis = _process_axis(Axis1, 0, workspace)
     y_axis = _process_axis(Axis2, 1 if workspace.is_PSD else 2, workspace)
+    return app.MAIN_WINDOW.slice_plotter_presenter.create_slice(workspace, x_axis, y_axis, None, None, NormToOne,
+                                                                'viridis')
 
-    return mantid_algorithms.Slice(InputWorkspace=workspace, XAxis=x_axis.to_dict(), YAxis=y_axis.to_dict(),
-                                   EMode=workspace.e_mode, PSD=workspace.is_PSD, NormToOne = NormToOne)
 
 def Cut(InputWorkspace, CutAxis=None, IntegrationAxis=None, NormToOne=False):
     """ Cuts workspace.
-
      Keyword Arguments:
-    InputWorkspace -- Workspace to cut. The parameter can be either a python handle to the workspace
-    OR the workspace name as a string.
+    InputWorkspace -- Workspace to cut. The parameter can be either a python
+                      handle to the workspace OR the workspace name as a string.
 
     CutAxis -- The x axis of the cut. If not specified will default to |Q| (or Degrees).
     IntegrationAxis --  The integration axis of the cut. If not specified will default to DeltaE.
     Axis Format:-
-           Either a string in format '<name>, <start>, <end>, <step_size>' e.g. 'DeltaE,0,100,5' (step_size may be
-           omitted for the integration axis) or just the name e.g. 'DeltaE'. In that case, the start and end will
-           default to the range in the data.
+            Either a string in format '<name>, <start>, <end>, <step_size>' e.g.
+            'DeltaE,0,100,5' (step_size may be omitted for the integration axis)
+            or just the name e.g. 'DeltaE'. In that case, the start and end will
+            default to the full range of the data.
 
     NormToOne -- if true the cut will be normalized to one.
 
     """
     _validate_workspace(InputWorkspace)
     workspace = get_workspace_handle(InputWorkspace)
-    if workspace.is_PSD:
-        if isinstance(workspace, MatrixWorkspace):
-            raise RuntimeError("Incorrect workspace type - run MakeProjection first.")
-        if not isinstance(workspace, PixelWorkspace):
-            raise RuntimeError("Incorrect workspace type.")
-    else:
-        if not isinstance(workspace, MatrixWorkspace):
-            raise RuntimeError("Incorrect workspace type.")
-
+    _check_workspace_type(workspace)
     cut_axis = _process_axis(CutAxis, 0, workspace)
     integration_axis = _process_axis(IntegrationAxis, 1 if workspace.is_PSD else 2,
                                      workspace, string_function=_string_to_integration_axis)
 
     return mantid_algorithms.Cut(InputWorkspace=workspace, CutAxis=cut_axis.to_dict(),
                                  IntegrationAxis=integration_axis.to_dict(), EMode=workspace.e_mode,
                                  PSD=workspace.is_PSD, NormToOne=NormToOne)
+
+
+def PlotSlice(InputWorkspace, IntensityStart="", IntensityEnd="", Colourmap='viridis'):
```

Comment:
Can we centralize the default colourmap selection? Maybe in `models/cmap`? If we wanted to change the default then I currently have to change it in a few different places.